tests/integration/run_test.go

```diff
@@ -27,8 +27,10 @@ func runTests(t *testing.T, when spec.G, it spec.S) {
 			assert.Equal(t, 1, service.GetAvailableReplicas(), "should have one available replica")
 			assert.Equal(t, 100, service.GetCurrentWeight())
 			assert.Equal(t, "nginx", service.GetImage())
-			assert.Contains(t, runningPods, service.App.Name)
-			assert.Contains(t, runningPods, "2/2")
+			for _,pod := range runningPods {
```

Comment:
I changed the `runningPods` function to return an array of the running pods, so now it will essentially returns something like:
```
[
"hi-6c9597ddb9-rl4nk   2/2     Running   0          28m",
"hi-6c9597ddb9-zjkwk   2/2     Running   0          34s",
"hi-6c9597ddb9-ct9mt   2/2     Running   0          34s"
]
```
...which is more useful to have because now we can validate that each pod retrieved has the service name and is running with 2/2. (Essentially, the scenario in this case is: "when we scale the service, all pods should be available")

Reply:
Hmmm, I suppose so. Feels like you are testing Kubernetes itself now though

Reply:
Maybe.. I think it's rio's flavor though, since it needs each pod to have both the linkerd proxy container and the app container to function properly